.github/workflows/publish-technical-documentation-next.yml

```diff
@@ -0,0 +1,36 @@
+name: "publish-technical-documentation-next"
+
+on:
+  push:
+    branches:
+    - "main"
+    paths:
+    - "docs/sources/**"
+jobs:
+  test:
+    runs-on: "ubuntu-latest"
+    steps:
+    - name: "Check out code"
+      uses: "actions/checkout@v3"
+    - name: "Build website"
+      run: |
+        docker run -v ${PWD}/docs/sources:/hugo/content/docs/mimir/latest -e HUGO_REFLINKSERRORLEVEL=ERROR --rm grafana/docs-base:latest /bin/bash -c 'make hugo'
+
+  sync:
+    runs-on: "ubuntu-latest"
+    needs: "test"
+    steps:
+    - name: "Clone website-sync Action"
+      if: "steps.has-matching-release-tag.outputs.bool == 'true'"
```

Comment:
I think this `if` condition needs to be removed.

Reply:
Yup, you're right [95700d3](https://github.com/grafana/mimir/pull/1820/commits/95700d3aca10c4b5177b76377296e48ae442fd78)